docs/UserGuide.md

```diff
@@ -115,14 +115,16 @@ Examples:
 Adds a new meeting to MyInsuRec. 
 DATE should be in DDMMYYYY format and TIME should be in 24-hour format.
 
-Format: `addMeeting n/NAME d/DATE t/TIME`
+Format: `addMeeting i/INDEX d/DATE t/TIME`
```

Comment:
Will also require a description field for this actually

Reply:
okay  :+1: